src/components/molecules/NavBar/NavBar.tsx

```diff
@@ -110,15 +119,29 @@ const NavBar: React.FC<NavBarPropsType> = ({ redirectionUrl }) => {
 
   const hasUpcomingEvents = futureUpcoming && futureUpcoming.length > 0;
 
+  const hasRadioStations = radioStations && radioStations.length;
+  const isSoundCloud =
+    !!hasRadioStations && RegExp("soundcloud").test(radioStations![0]);
```

Comment:
We could do this like `radioStations?.[0]?.includes('soundcloud')` or similar I think. A more robust way would be to parse the URL and check the domain host/etc. But for now, this is fine.